api/appsDev.ts

```diff
@@ -16,3 +19,12 @@ export async function fetchPublicAppsForPortal(
 
   return resp ? resp.results : [];
 }
+
+export function fetchPublicAppDeveloperTestAccountInstallData(
+  appId: number,
+  accountId: number
+): Promise<PublicAppDeveloperTestAccountInstallData> {
+  return http.get<PublicAppDeveloperTestAccountInstallData>(accountId, {
```

Comment:
I know this is a very simple method, but I think a unit test or two to verify the contract might be beneficial.